CHANGELOG.md

```diff
@@ -65,7 +65,7 @@
 
 
 ### 🛡 Security
-
+- Fix cve issue for transitive dependency @babel/runtime-corejs2 ([1531](https://github.com/opensearch-project/oui/pull/1531))
```

Comment:
These are changes for 1.17 - this should be go in the unreleased section, right?